test/test-start.js

```diff
@@ -0,0 +1,37 @@
+var assert = require("assert"); // assert module is supported from nodejs.

```

Comment:
FIXME(Wuseok): babel is not build for test module.
Current babel was built in server part so It'll fix later.